app/src/main/res/drawable/ic_alert.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#FFFFFF"
```

Comment:
Si llegaste hasta aquí, alegría!!! de aquí para abajo no hace falta mirar más, son iconos y los xml que se ven mejor directamente desde el studio, si algo te chirría comentame 😄 

https://gph.is/18Bev0r

Reply:
tenemos que aprender a poner gifs en las PR son geniales